breathecode/assignments/serializers.py

```diff
@@ -211,6 +212,24 @@ def validate(self, data):
                     'review status',
                     slug='editing-revision-status-but-is-not-teacher-or-assistant')
 
+        if 'opened_at' in data and data['opened_at'] > self.instance.opened_at:
```

Comment:
if you wanna do this in the serializer, should be better placed it in the create or update method, this method just run the validation, this shouldn't be runned if it's runned in bulk mode, and one of them fail its checks, this endpoint I think does not have bulk mode support but this should break in a future